is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
```

Comment:
What's the time and space complexity? Can we improve either of them?
```suggestion
    if (array.slice(currentIndex + 1).includes(currentElement)) {
  isAllUnique = false;
}
```